src/caret_analyze/infra/interface.py

```diff
@@ -40,7 +41,7 @@ def callback_records(
 
         Parameters
         ----------
-        callback_info : CallbackStructInfo
+        callback_info : CallbackStructValueInfo
```

Comment:
ここは元の実装が誤っていました。
実装と揃えてください。

```suggestion
        callback_info : CallBackStructValue
```

Reply:
対応しました。